tools/core_memory_recall.py

```
@@ -155,6 +157,33 @@ async def run(
                 )
                 for mem in sampled_memories
             ]
+
+            # 注入跨群聊画像（top-K 语义过滤 + 优先级标记）
+            if self.injector:
+                try:
+                    user_id = event.get_sender_id()
+                    profile_tags = self.injector.read_profile_tags(str(user_id))
+                    if profile_tags and query:
+                        # 按 query 关键词匹配排序，只注入相关画像
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)